tests/network-setups/fabric/dev/scripts/deployCC.sh

```diff
@@ -271,14 +271,16 @@ chaincodeInvokeInit() {
   # peer (if join was successful), let's supply it directly as we know
   # it using the "-o" option
   set -x
-  if [ "$CC_CHAIN_CODE" = "snamcc" ] ; then
-  	peer chaincode invoke -o localhost:${ORD_P} --ordererTLSHostnameOverride orderer.$NW_NAME.com --tls $CORE_PEER_TLS_ENABLED --cafile $ORDERER_CA -C $CHANNEL_NAME -n $CC_CHAIN_CODE $PEER_CONN_PARMS --isInit -c '{"function":"init","Args":["fnimcc"]}' >&log.txt
-  elif [ "$CC_CHAIN_CODE" = "fnimcc" ]; then
-        peer chaincode invoke -o localhost:${ORD_P} --ordererTLSHostnameOverride orderer.$NW_NAME.com --tls $CORE_PEER_TLS_ENABLED --cafile $ORDERER_CA -C $CHANNEL_NAME -n $CC_CHAIN_CODE $PEER_CONN_PARMS --isInit -c '{"function":"init","Args":[]}' >&log.txt
-  elif [ "$CC_CHAIN_CODE" = "interop" ]; then 
+  if [ "$CC_CHAIN_CODE" = "interop" ]; then
         peer chaincode invoke -o localhost:${ORD_P} --ordererTLSHostnameOverride orderer.$NW_NAME.com --tls $CORE_PEER_TLS_ENABLED --cafile $ORDERER_CA -C $CHANNEL_NAME -n $CC_CHAIN_CODE $PEER_CONN_PARMS --isInit -c '{"function":"initLedger","Args":["appId"]}' >&log.txt
-  else
+  elif [ "$CC_CHAIN_CODE" = "simplestate" ]; then
```

Comment:
We can leave this here for now, but in the future we should refactor these scripts to separate interop module deployment from application deployment.